exercises/exercise_5/README.md

```diff
@@ -108,66 +115,91 @@ To update the vote and results services, let's do the following:
+    j24jm0lyp8de        voting-app_worker    replicated          1/1                 dockersamples/examplevotingapp_worker:latest   
     ```
+    </details>
 
-3. In the repo's directory, start _only_ the `vote` service in the `docker-compose.yml` file. You should see it start up and the port badge for `5000` appear. Once it appears, click on it to validate the `vote` service has started.
+3. Deploy the `vote` service as a standalone container, attached to the `frontend` network with `OPTION_A` and `OPTION_B` environment variable set to `Cats` and `Dogs`. You should see the port badge for `5000` appear on the master node. Once it appears, click on it to validate the `vote` service has started.
+
+    <details>
+      <summary>Solution</summary>
 
     ```console
-    $ docker-compose up -d vote
-    Compose does not use swarm mode to deploy services to multiple nodes in a swarm. All containers will be scheduled on the current node.
+    $ docker --context swarm container run --name voting-app-vote-svc-dev -d --rm -e OPTION_A=Cats -e OPTION_B:Dogs -p 5000:80 --network front-tier mikesir87/examplevotingapp_vote
+    ```
+    <details>
 
-    To deploy your application across the swarm, use `docker stack deploy`.
+4. Clone the voting app repo (found at github.com/mikesir87/example-voting-app).
 
-    Creating example-voting-app_vote_1 ... done
+    ```console
+    $ git clone https://github.com/mikesir87/example-voting-app.git
     ```
 
-4. Now, let's make a change to the application! In the `vote/templates/index.html` file, let's make a change. You can change anything, but one idea might be to adjust the "Tip: you can change your vote" text to say "Tip: try changing your vote!". After saving the change, refresh the page displaying the vote app and you should see the change.
+5. Now, let's make a change to the application! In the `vote/templates/index.html` file, let's make a change. You can change anything, but one idea might be to adjust the "Tip: you can change your vote" text to say "Tip: try changing your vote!". After saving the change, refresh the page displaying the vote app and you should see the change.
 
-5. Now that we've made the change we want to save in our app, let's stop the dev container.
+6. Now that we've made the change we want to save in our app, let's stop the dev container.
 
     ```console
-    $ docker-compose down
-    Stopping example-voting-app_vote_1 ... done
-    Removing example-voting-app_vote_1 ... done
+    $ docker container stop $(docker container ls --filter name=voting-app-vote-svc-dev -q)
     ```
 
-6. Let's build our new image and push it to Docker Hub. Tag the image using `[your-docker-hub-username]/updated-vote-app`.
+7. Let's build our new image and push it to Docker Hub. Tag the image using `[your-docker-hub-username]/updated-vote-app`.
 
     ```console
     $ cd vote
-    $ docker build -t mikesir87/updated-vote-app .
+    $ docker build -t <your-docker-hub-username>/updated-vote-app .
```

Comment:
docker app build here?

Reply:
no, it's for building the vote service image

Reply:
Yes but `docker app build` does it